commands.c

```diff
@@ -298,50 +298,285 @@ show_help(int argc, char **argv)
+};
+
+struct ghs maxqueuetab[] = {
+	{ "<number>", "Maximum unassembled IP fragments in the fragment queue", CMPL0 NULL, 0 },
+	{ NULL, NULL, NULL, NULL, 0 }
+};
+
+struct ghs mtudisctab[] = {
+	{ "0", "Disable Path MTU Discovery", CMPL0 NULL, 0 },
+	{ "1", "Enable Path MTU Discovery", CMPL0 NULL, 0 },
+	{ NULL, NULL, NULL, NULL, 0 }
+};
+
+struct ghs mtudisctimeouttab[] = {
+	{ "<seconds>", "Timeout in seconds for routes added by Path MTU discovery engine", CMPL0 NULL, 0 },
+	{ NULL, NULL, NULL, NULL, 0 }
+};
+
+struct ghs ipsectimeouttab[] = {
+	{ "<seconds>", "Seconds after a SA is established before it will expire", CMPL0 NULL, 0 },
+	{ NULL, NULL, NULL, NULL, 0 }
+};
+
+struct ghs ipsecsofttimeouttab[] = {
+	{ "<seconds>", "Seconds after a SA is established before being renegotiated", CMPL0 NULL, 0 },
+	{ NULL, NULL, NULL, NULL, 0 }
+};
+
+struct ghs ipsecallocstab[] = {
+	{ "<number>", "Maximum IPSEC flows that can use a SA before it expires", CMPL0 NULL, 0 },
+	{ NULL, NULL, NULL, NULL, 0 }
+};
+
+struct ghs ipsecsoftallocstab[] = {
+	{ "<number>", "Maximum IPSEC flows that can use a SA before being renegotiated", CMPL0 NULL, 0 },
+	{ NULL, NULL, NULL, NULL, 0 }
+};
+
+
+struct ghs ipsecbytestab[] = {
+	{ "<number>", "Maximum bytes processed by a security association before it expires", CMPL0 NULL, 0 },
+	{ NULL, NULL, NULL, NULL, 0 }
+};
+
+struct ghs ipsecsoftbytestab[] = {
+	{ "<number>", "Maximum bytes processed by a security association before being renegotiated", CMPL0 NULL, 0 },
+	{ NULL, NULL, NULL, NULL, 0 }
+};
+
+struct ghs ipsecexpireacquiretab[] = {
+	{ "<seconds>", "Seconds the kernel allows to dynamically acquire SAs before a request", CMPL0 NULL, 0 },
+	{ NULL, NULL, NULL, NULL, 0 }
+};
+
+struct ghs ipsecfirstusetab[] = {
+	{ "<seconds>", "Seconds after security association is first used before it expires", CMPL0 NULL, 0 },
+	{ NULL, NULL, NULL, NULL, 0 }
+};
+
+
+struct ghs ipsecsoftfirstusetab[] = {
+	{ "<seconds>", "Seconds after a SA is first used before it is sent for renegotiation", CMPL0 NULL, 0 },
+	{ NULL, NULL, NULL, NULL, 0 }
+};
+
+struct ghs ipsecinvalidlifetab[] = {
+	{ "<seconds>", "Lifetime of Embryonic SAs in seconds", CMPL0 NULL, 0 },
+	{ NULL, NULL, NULL, NULL, 0 }
+};
+
+struct ghs ipsecpfstab[] = {
+	{ "0", "Disable Perfect Forward Secrecy", CMPL0 NULL, 0 },
+	{ "1", "Enable Perfect Forward Secrecy", CMPL0 NULL, 0 },
+	{ NULL, NULL, NULL, NULL, 0 }
+};
+
+struct ghs portfirsttab[] = {
+	{ "<number>", "Minimum registered port number for TCP/UDP port allocation", CMPL0 NULL, 0 },
+	{ NULL, NULL, NULL, NULL, 0 }
+};
+
+struct ghs porthifirsttab[] = {
+	{ "<number>", "Minimum dynamic/private port number for TCP/UDP port allocation", CMPL0 NULL, 0 },
+	{ NULL, NULL, NULL, NULL, 0 }
+};
+
+struct ghs porthilasttab[] = {
+	{ "<number>", "Maximum dynamic/private port number for TCP/UDP port allocation", CMPL0 NULL, 0 },
+	{ NULL, NULL, NULL, NULL, 0 }
+};
+
+struct ghs portlasttab[] = {
```

Comment:
lines 519-522  move this block to start  509  so it is paired up with portfirsttab []